sdk/storage/azure-storage-blob/azure/storage/blob/_shared/request_handlers.py

```diff
@@ -137,13 +137,18 @@ def validate_and_format_range_headers(
     return range_header, range_validation
 
 
-def add_metadata_headers(metadata=None):
-    # type: (Optional[Dict[str, str]]) -> Dict[str, str]
-    headers = {}
-    if metadata:
-        for key, value in metadata.items():
-            headers[f'x-ms-meta-{key.strip()}'] = value.strip() if value else value
-    return headers
+def add_metadata_headers(metadata: Optional[Dict[str, str]] = None, **kwargs: Dict[str, Any]) -> Dict[str, str]:
```

Comment:
TODO: Revert this change - `x-ms-` is required to be lowercased. This is ultimately up to the service.